### ISSUE TITLE
Issue with Indentation check and indentation after line wrapping

### ISSUE THREAD

**Issue by liutikas** [checkstyle/checkstyle#3460]
$ javac Test.java
`[no output]`

$ cat Test.java
```
public class Test {
    private static final int THIS_IS_THE_BEST_CONSTANT_OF_ALL_TIMES = 1;
    public boolean fooBar(int verylongparametername) {
        return java.lang.Math.max(
                THIS_IS_THE_BEST_CONSTANT_OF_ALL_TIMES, verylongparametername)
            == THIS_IS_THE_BEST_CONSTANT_OF_ALL_TIMES;
    }
}
```

$ cat config.xml
```
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="Indentation">
      <property name="severity" value="error"/>
      <property name="basicOffset" value="4"/>
      <property name="throwsIndent" value="8"/>
      <property name="lineWrappingIndentation" value="8"/>
      <property name="forceStrictCondition" value="true"/>
    </module>
  </module>
</module>
```

$ java -jar checkstyle-7.1.1-all.jar -c config.xml Test.java
```
Starting audit...
Audit done.

```

---------------
I expected Indentation check to catch the issue with the indention of  the line with ==, but it did not warn about anything.
---------------

It happens on Checkstyle 7.1.1 as well as older versions like 6.12.1.

**Comment by liutikas** [comment]
It seems to fail anytime line wrap happens between method parenthesis on a return statement.

Example 1:
```
public class Test {
    public boolean fooBar(int verylongparametername) {
        return java.lang.Math.max(
verylongparametername, 0);
    }
}
```

Example 2:
```
public class Test {
    public boolean fooBar(int verylongparametername) {
        return foo()
                + bar(
verylongparametername);
    }
}
```

**Comment by liutikas** [comment]
I guess it does not even have to be a return statement. It fails on the following as code well:
```
public class Test {
    public boolean fooBar(int verylongparametername) {
        foo() + foo(
verylongparametername);
    }
}
```

but it **does not** fail (it caches the issue) in:
```
public class Test {
    public boolean fooBar(int verylongparametername) {
        foo(
verylongparametername);
    }
}
```
